docs/develop/enable-nested-app-authentication-in-your-add-in.md

```diff
@@ -15,14 +15,21 @@ You can use the MSAL.js library (version 3.11 and later) with nested app authent
 - You can use incremental and dynamic consent for scopes.
 - You don't need to [preauthorize your hosts](/microsoftteams/platform/m365-apps/extend-m365-teams-personal-tab?tabs=manifest-teams-toolkit#update-azure-ad-app-registration-for-sso) (For example, Teams, Office) to call your endpoints.
 
+## NAA supported accounts and hosts
+
 > [!IMPORTANT]
 > Nested app authentication (NAA) is currently in preview. To try this feature, join the Microsoft 365 Insider Program (https://insider.microsoft365.com/join) and choose the Beta Channel. Don't use NAA in production add-ins. We invite you to try out NAA in test or development environments and welcome feedback on your experience through GitHub (see the **Feedback** section at the end of this page).
-> NAA is supported in the following builds.
->
-> - Word, Excel, PowerPoint, and classic Outlook on Windows Build 16.0.17531.20000 or later
-> - Word, Excel, PowerPoint, and Outlook on Mac Build 16.85.24040319 or later
->
-> Support for NAA in event-based activation code in Outlook is coming soon.
+
+NAA supports both Microsoft Accounts and Microsoft Entra ID (work/school) identities. It doesn't support Azure Active Directory B2C for business-to-consumer identity management scenarios. The following table explains the current support in preview. Anything listed as "Coming soon" will be supported by the time that NAA is made generally available.
+
+| Application | Windows                                                                            | Mac | Web          | iOS/iPad   | Android        |
+|-------------|------------------------------------------------------------------------------------|-----|--------------|------------|----------------|
+| Excel       | Yes (Current Channel (Preview))                                                    | Yes | Coming soon  | Yes (iPad) | Not applicable |
+| Outlook     | Yes* (Current Channel (Preview) for classic Outlook only, new Outlook coming soon) | Yes | Yes          | Yes (iOS)  | Not applicable |
```

Comment:
Originally, Android was "Yes" on Outlook. Just checking if it should still say that.

```suggestion
| Outlook     | Yes* (Current Channel (Preview) for classic Outlook only, new Outlook coming soon) | Yes | Yes          | Yes (iOS)  | Yes            |
```